src/shared/components/common/loading-skeleton.tsx

```diff
@@ -0,0 +1,127 @@
+import { Component } from "inferno";
+
+interface LoadingSkeletonProps {
+  itemCount?: number;
+}
+
+interface LoadingSkeletonLineProps {
+  size: 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+}
+
+class LoadingSkeletonLine extends Component<LoadingSkeletonLineProps, any> {
+  render() {
+    const className = "placeholder placeholder-lg col-" + this.props.size;
+    return (
+      <p className="placeholder-glow m-0">
+        <span className={className} style={{ height: "1.3em" }}></span>
```

Comment:
Nitpick: this can be a self-closing tag
```suggestion
        <span className={className} style={{ height: "1.3em" }} />
```